runtime/src/bank/check_transactions.rs

```diff
@@ -252,14 +222,7 @@ impl Bank {
         let nonce_data =
             nonce_account::verify_nonce_account(&nonce_account, message.recent_blockhash())?;
 
-        let nonce_is_authorized = message
-            .get_ix_signers(NONCED_TX_MARKER_IX_INDEX as usize)
-            .any(|signer| signer == &nonce_data.authority);
-        if !nonce_is_authorized {
-            return None;
-        }
```

Comment:
this removal ensures single-batching and entry-batching behave the same, it is consensus-safe without a feature gate because the distinction only matters after simd83